src/fragments/lib/storage/flutter/list.mdx

```diff
@@ -53,16 +33,21 @@ Future<void> listAlbum() async {
 
 ```
 
-Pagination is enabled by default. The default `pageSize` is `1000` if it has not been defined in the `S3ListOptions`. You can also list all files under a given path without pagination by using the `listAll` options constructor.
+Pagination is enabled by default. The default `pageSize` is `1000` if it is not set in the `StorageListOptions`.
+
+You can also list all files under a given path without pagination by using the `pluginOptions` and `S3ListPluginOptions.listAll()` constructor.
 
 This will list all public files (i.e. those with `guest` access level):
 
 ```dart
 Future<void> listAllWithGuestAccessLevel() async {
   try {
     final result = await Amplify.Storage.list(
-      options: const S3ListOptions.listAll(
+      options: const StorageListOptions(
         accessLevel: StorageAccessLevel.guest,
+        pluginOptions: S3ListPluginOptions.listAll(
+          excludeSubPaths: true,
```

Comment:
`excludeSubPaths: true` lists only the files at the root level, should remove it from this particular example as "This will list **all** public files".